main.cpp

```diff
@@ -574,31 +575,31 @@ struct Tractor
     Headlight headlights;
     
     // Drive
-    void drive(Engine engine, GearBox gearBox);
+    void drive(float distance, float speed);
     // Reverse
-    void Reverse(Engine engine, GearBox gearBox);
+    void Reverse(float distance, float speed);
     // Turn on lights
-    void turnOnLights(Headlight headlights);
+    void turnOnLights(float initialIntensity);
 };
 
-void Tractor::drive(Engine engineA, GearBox gearBoxA)
+void Tractor::drive(float distance, float speed)
 {
-    gearBoxA.increaseTorque(1);
-    engineA.propelVehicle(100, 30);
+    gearBox.increaseTorque(1);
+    engine.propelVehicle(distance, speed);
 }
 
-void Tractor::Reverse(Engine engineA, GearBox gearBoxA)
+void Tractor::Reverse(float distance, float speed)
```

Comment:
member functions should be spelled beginning with a lowercase letter.
- [ ] revise